### ISSUE TITLE
what's the mean of "In-sample" and "Out-sample"?

### ISSUE THREAD

**Issue by HandleCoding** [jsyoon0823/GANITE#3]
Does in-sample refer to the results of the test set part of the observed data and out-sample refers to the results of the unobserved data? Looking forward to your answer.

**Comment by jsyoon0823** [comment]
Thank you for your interest in our paper.

Yes. In-sample means the training data. Out-sample means the testing data.
During training, we only see the outcome of a certain action. Thus, still in-sample metrics are meaningful in terms of PEHE.


Thanks!
